markdown/usage/troubleshooting.md

```diff
@@ -24,6 +28,81 @@ subtitle: How to troubleshoot common mistakes and issues
 - [Using a local version of iGenomes](#using-a-local-version-of-igenomes)
 - [Extra resources and getting help](#extra-resources-and-getting-help)
 
+
+## Troubleshooting basics
+
+These are the recommended steps for troubleshooting a pipeline. 
+
+### Start small
+
+Before using the pipeline with your data, make sure to run a test using: 
+
+```bash
+nextflow run nf-core/<pipeline_name> -profile test,docker
+```
+
+If Docker is not installed, you can replace `docker` with `singularity` or `conda`, see
+the [Getting Started](https://nf-co.re/usage/introduction) tutorial for further information. If a test fails, it might indicate that 
+there is an issue with installation or configuration rather than a pipeline error.
+
+You might also want to check the following:
+
+1. Nextflow is up to date. Use `nextflow self-update` to update a typical installation or `conda update nextflow` for a Bioconda installation.
+2. There is enough disk space, this will avoid running out of space while you are running the pipeline. 
+3. Docker daemon is running (if you are using Docker to manage dependencies).
+
+
+### Categorize the type of error
+
+For this step you try to identify when the error occurs:
+
+1. Before the first process.
+2. During the first process. 
```

Comment:
```suggestion
2. During the first process: when an error appears during the first process it might indicate an issue with software dependencies, to specify how Nextflow should handle dependencies you need to select a [configuration profile](https://nf-co.re/usage/configuration#basic-configuration-profiles). This type of error might also be related to a missing command required to run the pipeline. Example error:

    ```bash
    Command exit status: 
      127 
    Command output: 
      (empty) 
    Command error: 
      .command.sh: line 3: rsem-prepare-reference: command not found 
    Work dir: 
      /home/lfaller/nextflow/rnaseq/work/f7/b6ef5a3f12f5efbf641f19046aca74 
    Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script 
file named `.command.sh` 
    Unexpected error [AbortedException] 
     -- Check script '/home/lfaller/.nextflow/assets/nf-core/rnaseq/./workflows/rnaseq.nf' at line: 603 or see '.nextflow.log' file for more details
    ```
```